internal/mode/static/state/graph/httproute.go

```diff
@@ -172,16 +169,21 @@ func buildRoute(
 	ghr *v1.HTTPRoute,
 	gatewayNsNames []types.NamespacedName,
 ) *Route {
-	sectionNameRefs := buildSectionNameRefs(ghr.Spec.ParentRefs, ghr.Namespace, gatewayNsNames)
+	r := &Route{
+		Source: ghr,
+	}
+	sectionNameRefs, err := buildSectionNameRefs(ghr.Spec.ParentRefs, ghr.Namespace, gatewayNsNames)
+	if err != nil {
+		r.Valid = false
+		r.Conditions = append(r.Conditions, staticConds.NewRouteUnsupportedValue(err.Error()))
```

Comment:
In this case, because r.ParentRefs  will be  empty, the status will not be really reported -- no parent Refs of our Gateway will be set
- https://github.com/nginxinc/nginx-gateway-fabric/blob/6becde2d07d89a9cbb83e623dddf7488ea171f81/internal/mode/static/build_statuses.go#L39
- https://github.com/nginxinc/nginx-gateway-fabric/blob/6becde2d07d89a9cbb83e623dddf7488ea171f81/internal/framework/status/httproute.go#L27


I wonder if we need to set any conditions here at all? Because it will not be reported

Reply:
That makes sense, thanks!